### DISCUSSION TITLE
unblock BBC UHD 216p and add new channels ?

### DISCUSSION THREAD

**Discussion by O77T** [iptv-org/community#125] [Q&A] [answered]
How do you unblock this UHD BBC channels in IPTV from this  https://iptv-org.github.io/iptv/index.m3u link ? 

Can I request channels as well?
<img width="1680" alt="Screen Shot 2022-04-23 at 5 28 12 PM" src="https://user-images.githubusercontent.com/79104107/164946548-119c606c-5fb7-493c-bee1-4a37df35599c.png">

**Comment by BellezaEmporium** [comment, marked as answer]
There's no "way" to unblock them apart from using a VPN redirected to United Kingdom.

Yes, you can request channels, but you need to make an issue and to respect the [Community Guidelines](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#issue-reporting-guidelines)

**Reply by O77T** [reply]
thank you
